lib/osf-components/addon/components/node-metadata-form/styles.scss

```diff
@@ -11,9 +11,10 @@
         color: $color-link-black;
     }
 
-    div:not(.resource-edit) {
+    div:not(.resource-edit),
+    div:not(.metadata-description-field) {
```

Comment:
I have tested this but can test again.

Reply:
Your suggestion results in the following similar to Brian Geiger's suggestion to change margin to padding. We chose during scrum to not make the change.

<img width="1631" alt="image" src="https://user-images.githubusercontent.com/82047646/217058301-b2fc3e1c-f7d3-4529-8c2b-1fa67401cca7.png">

Reply:
I think this can be addressed by adding a rule for 
```
.resource-edit div { 
    margin: 5px 0 15px;
}
```
since these divs were getting the margins before due to this rule above

Reply:
Ok, this in tandem works. Pushing changes shortly.